### ISSUE TITLE
Adjust the outside color of an e-mail (border) is not able anymore via interface in 5.1

### ISSUE THREAD

**Issue by stefan-franz** [mautic/mautic#13893]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1.29

### What browsers are you seeing the problem on?

Firefox

### What happened?

In the Vers. 5.0.4 i had anywhwere a color field right in the interface like for parts inside the e-mail body- but in the 5.1 i can’t find it.

I mean it's it a bug of the new behaviour of the section (section is now shown the full wide - at 5.0.4 it was shown only around the mail section)

### How can we reproduce this issue?

Note: Mautic Vers. 5.1 is missed in the select options here above (5.0 only is displayed)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
@stefan-franz for those of us not text-inclined, could you please provide us with some screenshots to demonstrate what you are referring to? That might help people to guide you more closely.

**Comment by stefan-franz** [comment]
In Mautic v 5.04  there was anywhere a way to adjust (in my way) the blue border color e.g. to green or something. After 5.1. the editor has changed and so i can't adjust it with the right toolbox.

The color field you see is the color of the first section - not the border color.

![Border Color not adjustable](https://github.com/user-attachments/assets/6ee3e101-3f39-4902-a337-86f1c01836a2)

**Comment by LordRembo** [comment]
This can be closed, it's a duplicate of #14127
